### PR TITLE
Add `instance_enabled` flag

### PR DESCRIPTION
**Pull request by SweetOps**
## What

* Add creation flag

## Why

* Possibility to disable instance if them doesn't need in current infrastructure

### CODE REVIEW

File: variables.tf [hunk 1]
```diff
@@ -28,6 +28,10 @@ variable "ansible_dry_run" {
   default = false
 }
 
+variable "create" {
```
Comment (osterman) [inline variables.tf]: let's call it `instance_count`

**Comment by osterman** [comment]
The label should add an attribute like this:
```
attributes = ["${var.instance_count > 1 ? var.instance_count : ""}"
```

**Comment by SweetOps** [comment]
screenshot 2017-09-22 12 17 30

File: main.tf [hunk 2]
```diff
@@ -19,25 +19,31 @@ data "aws_iam_policy_document" "default" {
...
-  source    = "git::https://github.com/cloudposse/tf_label.git?ref=tags/0.1.0"
-  namespace = "${var.namespace}"
-  stage     = "${var.stage}"
-  name      = "${var.name}"
+  source     = "git::https://github.com/cloudposse/tf_label.git?ref=tags/0.2.0"
+  namespace  = "${var.namespace}"
+  stage      = "${var.stage}"
+  name       = "${var.name}"
+  delimiter  = "${var.delimiter}"
+  attributes = ["${var.instance_count ? var.instance_count : ""}"]
```
Comment (comeanother) [inline main.tf]: Rename `instance_count` to something precise e.g. `create_instance`.

Or use `int` type instead of `boolean`.

File: main.tf [hunk 3]
```diff
@@ -19,20 +19,24 @@ data "aws_iam_policy_document" "default" {
 
 # Apply the tf_label module for this resource
 module "label" {
-  source    = "git::https://github.com/cloudposse/tf_label.git?ref=tags/0.1.0"
-  namespace = "${var.namespace}"
-  stage     = "${var.stage}"
-  name      = "${var.name}"
+  source     = "git::https://github.com/cloudposse/tf_label.git?ref=tags/0.2.0"
```
Comment (osterman) [inline main.tf]: fix module ref

File: main.tf [hunk 4]
```diff
@@ -19,20 +19,24 @@ data "aws_iam_policy_document" "default" {
...
-  source    = "git::https://github.com/cloudposse/tf_label.git?ref=tags/0.1.0"
-  namespace = "${var.namespace}"
-  stage     = "${var.stage}"
-  name      = "${var.name}"
+  source     = "git::https://github.com/cloudposse/terraform-null-label.git?ref=tags/0.2.1"
+  namespace  = "${var.namespace}"
+  stage      = "${var.stage}"
+  name       = "${var.name}"
+  delimiter  = "${var.delimiter}"
+  attributes = ["${var.instance_enabled ? var.instance_enabled : ""}"]
```
Comment (comeanother) [inline main.tf]: What `label.id` will have the instance?

**Comment by SweetOps** [comment]
screenshot 2017-09-26 14 26 11

screenshot 2017-09-26 14 33 13

**Comment by comeanother** [comment]
@SweetOps you should use different names for `namespace`, `stage` and `name`. Otherwise it is impossible to figure out is there a naming error or no.

**Pull request merged**